dbsnp-importer/src/test/java/uk/ac/ebi/eva/dbsnpimporter/configuration/ImportVariantsStepConfigurationTest.java

```diff
@@ -52,7 +59,7 @@
 @RunWith(SpringRunner.class)
 @TestPropertySource({"classpath:application.properties"})
 @ContextConfiguration(classes = {ImportEvaSubmittedVariantsJobConfiguration.class, MongoTestConfiguration.class,
```

Comment:
Change from ImportEvaSubmittedVariantsJobConfiguration to ImportsVariantsJobConfiguration